src/app/settings/organization-plans.component.ts

```diff
@@ -204,6 +208,17 @@ export class OrganizationPlansComponent implements OnInit {
             }
         }
 
+        if (this.onlyOrgPolicyBlock) {
+            return;
+        }
+        else {
+            const policies = await this.policyService.getAll(PolicyType.OnlyOrg);
+            this.onlyOrgPolicyBlock = policies.filter(policy => policy.enabled).length > 0;
+            if (this.onlyOrgPolicyBlock) {
+                return;
+            }
+        }
```

Comment:
formatting on if/else. Might want to do this first, before the file parsing above.

Reply:
fixed + moved